src/subscript/welltest_dpds/welltest_dpds.py

```diff
@@ -320,7 +327,7 @@ def weighted_avg_press_time_derivative_lag2(
     return dpdspt_weighted_lag2
 
 
-def to_csv(filen, field_list, header_list, start=0, end=None, sep=", "):
+def to_csv(filen, field_list, header_list=[None], start=0, end=None, sep=","):
```

Comment:
Never default to list objects, it will hide bugs. Do  `header_list=None` in the signature, and then

```
if header_list is None:
    header_list = [None]
```

Reply:
Ouch, good point. Fixed